README.rst

```diff
@@ -16,6 +16,20 @@ Features
 
 * TODO
 
+PyPI Package Release
+--------------------
+- This reposiroty publishes package on PyPI, when you update this repository make sure to create a new release so latest changes should be published on PyPI and become available for your use.
+- Bump the version `utils version`_ to match the release version, once pull reuqest with updated version is merged into master. Create a new release from GitHub Web UI. You can refer this document on how to `How to create GitHub release`_
+- `edx-prefectutils`_ package is used in `prefect-flows`_ repository, you will have to update `requirements`_ file version so it will pick latest PyPI package version.
```

Comment:
Oh, I was confused by this too. : )   I thought that we didn't specify a particular version in the requirements file, but instead had to update the flow image version in order to get a flow to rebuild.   So we also need to update the requirements *and* update the individual flow image version numbers?

Reply:
Yes correct and how to update image tag is documented in internal wiki page [](https://openedx.atlassian.net/wiki/spaces/DE/pages/1448673295/How+to+develop+prefect+flows#Updating-an-existing-flow)